src/GitVersionExe/HelpWriter.cs

```diff
@@ -47,6 +47,12 @@ public static void WriteTo(Action<string> writeAction)
                     it be created with these attributes: AssemblyFileVersion, AssemblyVersion and AssemblyInformationalVersion
                     ---        
                     Supports writing version info for: C#, F#, VB    
+
+    # Create or update Wix version file
+    /updatewixversionfile
```

Comment:
Does it make sense updating WiX files from GitVersionTask as well? If so, perhaps we should add a way to do that as well?

Reply:
For our use cases, changes to GitVersion exe suffice and while it is a valid use case but I don't aim to address GitVersionTask here.

Reply:
Ok.